django/authentication/Tests/test_views.py

```diff
@@ -0,0 +1,20 @@
+from .test_setup import TestSetUp
+
+class TestViews(TestSetUp):
+    def test_user_cannot_register_without_data(self):
+        res = self.client.post(self.register_url)
+        self.assertEqual(res.status_code , 400)
+        
+    def test_user_can_register_correctly(self):
+        res = self.client.post(
+            self.register_url, self.user_data, format="json")
+        #self.assertEqual(res.data['email'] ,self.user_data['email'])
```

Comment:
Were these temporarily commented out because they were failing? If so, I would suggest fixing the issue causing them to fail and uncommenting.

Reply:
Yes, I was having an import Issue. Sorting it out now